fe/src/components/RankAndStats/RankAndStats.re

```diff
@@ -1,48 +1,33 @@
 open EloTypes;
-open Js.Promise;
 
-open Svc;
 [%bs.raw {|require('./RankAndStats.scss')|}];
 
-type state =
-  | LOADING
-  | LOADED(list(user))
-  | FAILURE;
+let component = ReasonReact.statelessComponent("RankAndStats");
 
-let component = ReasonReact.reducerComponent("RankAndStats");
+module GetUsers = [%graphql
+  {|
+  query {
+    users @bsRecord {
+      userNid
+      code,
+      name,
+      rating,
+      team,
+      trendRatingDiff,
+    }
+  }
+|}
+];
 
-let initialState = () => LOADING;
+module GetUsersQuery = ReasonApollo.CreateQuery(GetUsers);
 
-let onSuccess = (send, json) =>
-  json |> DecodeUsers.users |> (users => send(SetUsersToState(users)));
+let refreshUsersWithRefetch = (refetch, ()) => refetch(None)->ignore;
```

Comment:
Wyjaśnisz mi, proszę, co oznacza `refetch(None)->ignore` ?

Reply:
refetch to funkcja dostarczona przez reason-apollo która ponownie wysyła request o pobranie danych.
Ona właśnie powoduje że wysyła się request i wtedy ten komponent wygenerowany z reason-apollo (GetUsersQuery) sam się odświży - i to chce osiągnąć. Ale ta funkcja dodaktowo zwraca promise, jeśli ktoś chciałby potem coś jeszcze robić. Ja nie chce i po to jest ten ignore - bo funkcja refreshUsersWithRefetch powinna zwrócić unita, a bez użycia ignore by zwróciła promise. Czyli w skrócie ignore to funkcja która bierze coś tam i zwraca unita.